spec/jekyll-admin/server/collection_spec.rb

```diff
@@ -32,56 +32,74 @@ def app
   end
 
   context "an individual collection" do
-    let(:documents) { last_response_parsed["documents"] }
-    let(:first_document) { documents.first }
-
     it "returns an individual collection" do
       get "/collections/posts"
       expect(last_response).to be_ok
       expect(last_response_parsed["label"]).to eq("posts")
     end
 
-    it "includes documents" do
+    it "doesn't includes documents" do
```

Comment:
Can you explain the changes to the API that are happening here and why? I'm having trouble following. If we're changing the API do the docs need to be updated?